source/common/upstream/BUILD

```diff
@@ -126,8 +126,10 @@ envoy_cc_library(
         "//source/common/http:utility_lib",
         "//source/common/network:filter_lib",
         "//source/common/protobuf:utility_lib",
-        "//source/common/redis:conn_pool_lib",
         "//source/common/router:router_lib",
+        # TODO(mattklein123): The following should not be required once we allow pluggable health
+        # checkers.
+        "//source/extensions/filters/network/redis_proxy:conn_pool_lib",
```

Comment:
Are you planning on doing this work?  If not, meta comment on TODOs, we may want to consider allowing TODO(#issuenumber) and filing tracking issues where one can code-search for relevant things to change.

Reply:
I wasn't immediately planning on doing it. I opened https://github.com/envoyproxy/envoy/issues/2933 to track. SGTM regarding TODO style.